packages/service-library/src/servicelib/background_task.py

```diff
@@ -29,7 +29,7 @@ async def _periodic_scheduled_task(
             ), log_catch(logger):
                 await task(**task_kwargs)
 
-            raise TryAgain
+            raise TryAgain()
```

Comment:
this change is not necessary

Reply:
I am aware of that, but consider the following: you are raising a class. Usually an exception is an instance of that class, it would make more sense to raise an instance instead of a class.